NEWS.md

```diff
@@ -1,6 +1,25 @@
 JuMP release notes
 ==================
 
+Version 0.20.0 (Aug 24, 2019)
+-----------------------------
+
+- Documentation updates.
+- Lots of bug fixes.
+- Better error messages (#2017).
+- Performance improvements (#2032).
+- Added LP sensitivity summary functions `lp_objective_perturbation_range`
+  and `lp_rhs_perturbation_range` (#1917).
+- Added functions `dual_objective_value`, `raw_status` and `set_parameter`.
+- Added function `set_objective_coefficient` to modify the coefficient of
+  a linear terms coefficient of the objective (#2008).
```

Comment:
s/the coefficient of a linear terms coefficient/the coefficient of a linear term/